config/200-source-resolvers-clusterrole.yaml

```diff
@@ -0,0 +1,74 @@
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Use this aggregated ClusterRole when you need readonly access to "Sources"
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: source-resolver
+  labels:
+    eventing.knative.dev/release: devel
+aggregationRule:
+  clusterRoleSelectors:
+  - matchLabels:
+      duck.knative.dev/source: "true"
+rules: [] # Rules are automatically filled in by the controller manager.
+
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: eventing-source-resolver
+  labels:
+    eventing.knative.dev/release: devel
+    duck.knative.dev/source: "true"
+# Do not use this role directly. These rules will be added to the "source-resolver" role.
+rules:
+- apiGroups:
+  - "sources.eventing.knative.dev"
+  resources:
+    - "cronjobsources"
+    - "containersources"
+    - "apiserversources"
+    - "awssqssources"
+    - "camelsources"
+    - "kafkasources"
+    - "githubsources"
+  verbs:
+  - get
+  - list
+  - watch
+---
+
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-gcp-source-resolver
+  labels:
+    eventing.knative.dev/release: devel
+    duck.knative.dev/source: "true"
+# Do not use this role directly. These rules will be added to the "source-resolver" role.
+rules:
+  - apiGroups:
+      - "pubsub.cloud.run"
+      - "events.cloud.run"
+    resources:
+      - "pullsubscriptions"
```

Comment:
Same for these 2 resources as the eventing-contrib ones. These should eventually be moved to config directories in https://github.com/google/knative-gcp.